docs/geedocs/answer/3230746.html

```diff
@@ -43,8 +43,8 @@
   <h2><a name="Introduction">Introduction</a></h2>
   <h3>Google Earth Enterprise Portable</h3>
   <p>Google Earth Enterprise Portable (GEE Portable) lets you view portable globes and maps on your laptop or desktop without requiring network access. This is useful for emergency responses to disasters like earthquakes or floods, or for maps that contain private information that you don’t want to share on the internet.</p>
-  <p>You can install GEE Portable on a Windows, Mac OS X, or Linux machine. It starts within seconds and automatically launches a web browser to display one of the portable globes or maps that's been saved to the GEE Portable <code>maps</code> directory.</p>
-  <div class="note"> <strong>Note:</strong> Instead of using GEE Portable, you can also connect to a portable globe or map with the Google Earth Enterprise Client (Google Earth EC) or the Google Earth Plug-in. </div>
+  <p>You can install GEE Portable on a Windows, or Linux machine. It starts within seconds, and, then, you can launch a web browser to display one of the portable globes or maps that's been saved to the GEE Portable <code>data</code> directory.</p>
```

Comment:
Well, a comma, indicates a, pause, when you read something outloud. So, maybe this, sentence, is meant to be, read like you're, William Shatner.